docs/publish/maintain-breaking-changes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Maintain your Office Add-in
+description: Understand our commitments to compatibility and how to keep your add-in up to date.
+ms.date: 04/29/2022
+ms.localizationpriority: medium
+---
+
+# Maintain your Office Add-in
+
+After you publish your add-in, you should keep it up to date with any important changes from upstream libraries. Patching security issues is critical to building customer trust. Since these changes have no effect on the published manifest, your customers won't need to take any actions to get the latest versions of your add-in.
+
+## Breaking changes in Office.js
+
+The Microsoft 365 Developer Platform is committed to ensuring the compatibility of your add-in. We strive to avoid making breaking changes to the API surface and behavior. However, there are cases were we need to make breaking updates for the sake of security or reliability. In those rare cases, the following steps are taken to ensure users of your add-in are unaffected.
```

Comment:
```suggestion
The Microsoft 365 Developer Platform is committed to ensuring the compatibility of your add-in. We strive to avoid making breaking changes to the API surface and behavior. However, there are cases where we need to make breaking updates for the sake of security or reliability. In those rare cases, the following steps are taken to ensure users of your add-in are unaffected.
```